Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -292,12 +277,11 @@ EmbeddedFrag::EmbeddedFrag(const RDKit::Bond *dblBond) {
 int EmbeddedFrag::findNumNeigh(const RDGeom::Point2D &pt, double radius) {
   // find the number of atoms in the current embedded system that are within
   // 'radius' of the specified point
-  INT_EATOM_MAP_CI efi;
   int res = 0;
-  for (efi = d_eatoms.begin(); efi != d_eatoms.end(); efi++) {
-    RDGeom::Point2D rloc = efi->second.loc;
+  for (const auto &efi : d_eatoms) {
+    RDGeom::Point2D rloc = efi.second.loc;
```

Comment:
This doesn't need to be a non-const copy, although I expect it really doesn't matter.

Reply:
good catch

Reply:
I think these are going to end up optimized into register anyway...